Lib/statistics.py

```diff
@@ -111,7 +111,7 @@
 from math import hypot, sqrt, fabs, exp, erf, tau, log, fsum
 from operator import itemgetter
 from collections import Counter
-
+from _statistics import _normal_dist_inv_cdf
```

Comment:
Just curious how does this work in environments where C accelerator might not be present? Is it better to have the Python definition in `statistics.py` as it is and then try to import `_statistics` to override `_normal_dist_inv_cdf` if available?

Reply:
@tirkarthi Great idea, by the way, do you know what I should do for windows platform when importing C module? I have only MacBook so I can't check it works well on windows.

cc @rhettinger @vstinner

Reply:
@tirkarthi 
IMHO, the current C implementation does not have a specific platform-dependent code.
So I think that it will work most of the platform which can compile `C`Python.
What do you think?

Reply:
Modules like datetime have C implementation and a pure Python implementation. I have seen PyPy using pure Python implementation. Mine was just a concern that do we need to make `statistics.py` dependent on C accelerator always for this helper. My concern might be wrong or I am misunderstanding other implementations shipping C code. I will wait for a core dev input on this.